x-pack/plugins/ml/common/types/annotations.ts

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+// The Annotation interface is based on annotation documents stored in the
+// `.ml-annotations-6` index, accessed via the `.ml-annotations` alias.
+
+// Annotation document mapping:
+// PUT .ml-annotations-6
+// {
+//   "mappings": {
+//     "annotation": {
+//       "properties": {
+//         "annotation": {
+//           "type": "text"
+//         },
+//         "create_time": {
+//           "type": "date"
+//         },
+//         "create_username": {
+//           "type": "keyword"
+//         },
+//         "timestamp": {
+//           "type": "date"
+//         },
+//         "end_timestamp": {
+//           "type": "date"
+//         },
+//         "job_id": {
+//           "type": "keyword"
+//         },
+//         "modified_time": {
+//           "type": "date"
+//         },
+//         "modified_username": {
+//           "type": "keyword"
+//         },
+//         "result_type": {
+//           "type": "keyword"
+//         }
+//       }
+//     }
+//   }
+// }
+
+// Alias
+// POST /_aliases
+// {
+//     "actions" : [
+//         { "add" : { "index" : ".ml-annotations-6", "alias" : ".ml-annotations" } }
```

Comment:
It would be good to create two aliases, `.ml-annotations-read` and `.ml-annotations-write`.  This will allow for there to be multiple underlying annotations indices in the future.

Reply:
Thanks for the feedback, I adjusted the comment and code for read/write aliases in 166a7cf.